SIMPLVtkLib/Visualization/Controllers/VSConcurrentImport.cpp

```diff
@@ -166,9 +167,13 @@ void VSConcurrentImport::importDataContainerArray(DcaGenericPair genericPair)
   // Wait for the filter lock
   m_FilterLock.acquire();
 
-  m_ThreadCountLock.acquire();
-  m_ThreadsRemaining = m_ThreadCount;
-  m_ThreadCountLock.release();
+  m_PartialWrappingThreadCountLock.acquire();
+  m_PartialWrappingThreadsRemaining = m_ThreadCount;
+  m_PartialWrappingThreadCountLock.release();
```

Comment:
I don't think there's time now, but you might want to look into std::atomic, as it's a lot more readable, concise, and harder to miss a mutex (un)lock.  You didn't mess that up, but it's slow to read and thus slow to maintain.

Reply:
Yeah, that looks interesting.  I was looking it up now and it looks like it's a lot less overhead and reads a lot better.